app/app.go

```diff
@@ -786,10 +786,10 @@ func NewExocoreApp(
 
 	(&app.EpochsKeeper).SetHooks(
 		epochstypes.NewMultiEpochHooks(
+			app.ExomintKeeper.EpochsHooks(),    // however, this may change once we have distribution
```

Comment:
Please update the comments after the keepers.

Reply:
updated